### ISSUE TITLE
mailto links not working?

### ISSUE THREAD

**Issue by sodevious** [datocms/vue-datocms#62]
Using the vue plugin for structured text like so: 
```
<datocms-structured-text :data="block.content" />
```

With the follwing data:

```
    "content": {
            "value": {
              "schema": "dast",
              "document": {
                "type": "root",
                "children": [
                  {
                    "type": "paragraph",
                    "children": [
                      {
                        "url": "mailto:test@tests.com",
                        "type": "link",
                        "children": [
                          {
                            "type": "span",
                            "value": "test@test.com"
                          }
                        ]
                      }
                    ]
                  }
                ]
              }
```

I get this rendered in the DOM:

```
<p><a>test@test.com</a></p>
```

**Comment by trampolinstudio** [comment]
@sistrall we think we have a same issue with two almost identical websites (copy of codebase) but one using vue-datocms 1.1.0 (nuxt 2.15.17) and the other using 4.0.2 (nuxt 2.15.18). The first one renders the links without the problem and the second one reproduces the issue above with all links.

Is there something more that we could provide to give you better context? Thank you!

**Comment by sistrall** [comment]
Hello @sodevious and @trampolinstudio ,

I managed to replicate the issue: the root cause of the problem is that Vue 2 and Vue 3 render function (usually called `h`) expect HTML attributes in different ways. So I published [version 4.0.5](https://github.com/datocms/vue-datocms/releases/tag/v4.0.5), which should solve the issue.
